pages/index.js

```diff
@@ -1432,20 +1432,18 @@ function onSearch(
         let deals = [];
 
         // Extract protocols and deal information
-        while (mdBytes.length > 0) {
+        if (mdBytes.length > 0) {
```

Comment:
Should we still display cases where multiple protocols are included / concatenated in the metadata?